libs/cangen/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "cangen"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+proc-macro2 = "1.0.89"
```

Comment:
add all of these dependencies as workspace dependencies in the main Cargo.toml and then reference them from here and daedelus Cargo.toml.  Ref; https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table